tests/AutoReview/FixerFactoryTest.php

```diff
@@ -772,6 +772,9 @@ private static function getFixersPriorityGraph(): array
             'phpdoc_line_span' => [
                 'no_superfluous_phpdoc_tags',
             ],
+            'phpdoc_list_type' => [
+                'phpdoc_types_order',
+            ],
```

Comment:
There seems to be more cases of priority dependencies in this PR, e.g. `phpdoc_scalar` must run before `phpdoc_list_type`. Why not adding those here?

Reply:
Because they are handled in [lines 958-979](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.48.0/tests/AutoReview/FixerFactoryTest.php#L958-L979). I've added 1 priority test and many PHPDocs with priorities had to change.